src/widget/form/settings/avform.cpp

```diff
@@ -65,6 +67,12 @@ AVForm::AVForm() :
     bodyUI->playbackSlider->installEventFilter(this);
     connect(bodyUI->playbackSlider, &QSlider::valueChanged,
             this, &AVForm::onPlaybackValueChanged);
+
+    bodyUI->microphoneSlider->setToolTip(
+                tr("Use slider to set the gain of your input device ranging"
+                   " from -30dB to +30dB."));
```

Comment:
These numbers should be retrieved from the defined constants currently in `Audio::Private`